msig_contract_deployment/msig_erc20_account_creation.md

```diff
@@ -0,0 +1,33 @@
+<h1>Multi-sig account creation of evm accounts for EOS-ERC20 token bridge </h1>
+
+<h2>Overview</h2>
+EOS-EVM will soon support trustless token bridge between tokens on native EOS (for example: USDT on EOS) and ERC-20 compatible tokens on EOS-EVM. (contract implementaion can be found in https://github.com/eosnetworkfoundation/evm-bridge-contracts). Because of such, we need the multi-sig from 21 block producers to create the following accounts on testnet & mainnet:
+
+<h3>account <b>eosio.erc2o</b></h3>
+
+  it is the account that handles the main logic of token bridge operations, such as token registration, token transfer between native EOS and EOS-EVM. The EVM ERC-20 bridge contract will be deployed into this account.
+
+<b>permission structure:</b>
+  - owner: set to eosio@active
+  - active: set to eosio.evm@active
+
+<b>list of actions:</b>
```

Comment:
I don't think we need this list of actions here. It just confused things.

This multisig does not actually deploy the contract. That will be another one (with its own separate document) that will be approved by 3 out of 5 BPs and the ENF. The document describing that multisig can provide some information about the actions. And even then I think it may be overkill, but I suppose it will be good to at least mention what the `setingressfee` and `withdrawfee` actions do since those are the ones that will be accessible from a custom permission.

But for this multi-sig which is only for the top 21 EOS BPs, there is no need to mention any of these actions.